src/app/src/pages/redteam/setup/components/Purpose.tsx

```diff
@@ -0,0 +1,180 @@
+import KeyboardArrowLeftIcon from '@mui/icons-material/KeyboardArrowLeft';
+import KeyboardArrowRightIcon from '@mui/icons-material/KeyboardArrowRight';
+import { Alert } from '@mui/material';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Grid from '@mui/material/Grid';
+import Stack from '@mui/material/Stack';
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+import { useRedTeamConfig } from '../hooks/useRedTeamConfig';
+import Prompts from './Prompts';
+
+interface PromptsProps {
+  onNext: () => void;
+  onBack: () => void;
+}
+
+export default function Purpose({ onNext, onBack }: PromptsProps) {
+  const { config, updateApplicationDefinition } = useRedTeamConfig();
+
+  const isPurposePresent = config.purpose && config.purpose.trim() !== '';
+
+  return (
+    <Stack direction="column" spacing={4}>
+      <Typography variant="h4" gutterBottom sx={{ fontWeight: 'bold', mb: 3 }}>
+        Application Details
+      </Typography>
+      <Alert severity="info">
+        The more information you provide, the better the redteam attacks will be and the more
+        accurate the results.
+      </Alert>
+
+      <Box>
+        <Stack direction="column" spacing={3}>
+          <Box>
+            <Typography variant="h5" gutterBottom sx={{ fontWeight: 'medium' }}>
+              Purpose
+            </Typography>
+            <Typography variant="body1">
+              This should describe the purpose of your application. Be as specific as possible.
+              Include information about who the user of the system is and what information and
+              actions they should be able to access.
+            </Typography>
+            <TextField
+              fullWidth
+              value={config.applicationDefinition.purpose}
+              onChange={(e) => updateApplicationDefinition('purpose', e.target.value)}
+              placeholder="e.g. You are a travel agent specialized in budget trips to Europe. The user is anonymous and should not be able to access any information about the system or other users."
```

Comment:
can we simplify this now that we've added the other fields?

```suggestion
              placeholder="e.g. You are a travel agent specialized in budget trips to Europe."
```